apps/demos/src/WalletSelector/SignMessage.tsx

```diff
@@ -0,0 +1,41 @@
+import { signMessage } from '@bos-web-engine/wallet-selector-plugin';
+import { Buffer } from 'buffer';
+import { useCallback, useState } from 'react';
+
+export default function () {
+  const [message, setMessage] = useState('');
+  const [recipient, setRecipient] = useState('');
+  const [signature, setSignature] = useState('');
+
+  const generateNonce = useCallback(() => {
```

Comment:
Looks like we could extract `generateNonce` to exist outside the component. This way we could avoid memoization with the `useCallback`.